app/main.py

```diff
@@ -1,8 +1,21 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> "Person":
+        self.name = name
+        self.age = age
+        self.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    for person in people:
+        Person(person["name"], person["age"])
+    for person in people:
+        if "wife" in person and person["wife"] is not None:
+            Person.people[person["name"]].wife = (
+                Person.people)[person["wife"]]
+            continue
```

Comment:
you anyway won't go in the next condition, so actually not necessary to add a continue here. I understand your logic, but it's not a common practice

Reply:
if you want I can remove it. but. without "continue" python will go to if statement and check the condition. this will be an unnecessary and avoidable step